action.yml

```diff
@@ -21,7 +21,7 @@ runs:
   using: 'node12'
   main: 'dist/restore/index.js'
   post: 'dist/save/index.js'
-  post-if: 'success()'
+  post-if: 'success() && !env.CACHE_SKIP_SAVE'
```

Comment:
Unfortunately this doesn't work well, as `env` values are always strings.

So for example both `CACHE_SKIP_SAVE: true` and `CACHE_SKIP_SAVE: false` would evaluate to a non-empty string ('true' and 'false`) which equal `true`.

See the debug logging in https://github.com/joshmgross/actions-testing/runs/1560134417?check_suite_focus=true

Reply:
Right!  
Thanks.  

How about that then -https://github.com/actions/cache/pull/489/commits/f4a5fe8db9227c95e5652eec8e065832f1b4b4d9